controller/src/main/java/org/jboss/as/controller/operations/common/XmlFileMarshallingHandler.java

```diff
@@ -36,7 +36,7 @@ public class XmlFileMarshallingHandler extends AbstractXmlMarshallingHandler {
             .setReplyParameters(new SimpleAttributeDefinitionBuilder(UUID, ModelType.STRING, false).build())
             .setReadOnly()
             .setRuntimeOnly()
-            .setStability(Stability.COMMUNITY)
+            .setStability(Stability.DEFAULT)
```

Comment:
I think we can remove the setStability call here, DEFAULT is the default when none is set so we can probably have one line less.